tests/Microsoft.AspNet.SignalR.Tests/Server/Hubs/TypedClientBuilderFacts.cs

```diff
@@ -145,6 +154,9 @@ public interface IAmGeneric<T>
         {
             void send(T genericArgument);
             void ping();
+            void send<U>(U u);
+            void send<U, V>(U u, V v);
+            void send<U, V>(U u, V v, U u1);
```

Comment:
I'm sure it will pass, but can we add a method that mixes and non-generic parameters?